app/src/main/java/com/example/mytrainingpal/components/StartExerciseFloatingButton.kt

```diff
@@ -2,13 +2,15 @@ package com.example.mytrainingpal.components
 
 import androidx.compose.material.FloatingActionButton
 import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
 import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.FitnessCenter
```

Comment:
Hey, I think these imports slipped in accidentally.
It looks like these are useless. Could you please check this and remove them if they are not needed?